### DISCUSSION TITLE
i would like to use webhook receiver for image update

### DISCUSSION THREAD

**Discussion by wibed** [fluxcd/flux2#4387] [General]
i thought about when an image updates and i would like to automate the rollout by:
- fetch when there are updates on the image, [https://fluxcd.io/flux/guides/image-update/](https://fluxcd.io/flux/guides/image-update/ "https://fluxcd.io/flux/guides/image-update/")
- run tests against the updated deployment (staging), https://docs.flagger.app/tutorials/istio-progressive-delivery
- if ok, lift it to production, else create error log and send mail 
  - receive ok, https://fluxcd.io/flux/guides/webhook-receivers/
  - chain next rollout, https://fluxcd.io/flux/guides/image-update/
  
  
TL:DR
Besides reconciliation, can I use the webhook receiver to chain the next rollout from staging to production given a conditional statement?
